ethcore/builtin/src/lib.rs

```diff
@@ -548,14 +572,39 @@ impl Bn128Pairing {
 	}
 }
 
+impl Implementation for EIP1962 {
+	/// Can fail in many cases, so leave it for an implementation
+	fn execute(&self, input: &[u8], output: &mut BytesRef) -> Result<(), &'static str> {
+		let result = eth_pairings::public_interface::API::run(&input);
+		if result.is_err() {
+			trace!("EIP1962 error: {:?}", result.err().expect("is error"));
+			return Err("Precompile call was unsuccessful");
```

Comment:
Can you include the actual error in the string?

Reply:
Sure, I was not sure if it would be handy